pkgs/applications/version-management/merge-fmt/default.nix

```diff
@@ -15,6 +15,9 @@ buildDunePackage rec {
 
   buildInputs = [ cmdliner base stdio ];
 
+  # core v0.17 compatibility, obtained by `git diff -r 3e37827~2..3e37827`
```

Comment:
There hasn't been a stable release of `merge-fmt`, but compatibility issues have been fixed in the tip of the Github repo. Perhaps instead of adding a patch file, we should just ditch the stable version and reference the tip in the `fetchurl` above? What should I use as the `version` in that case?